src/components/ingestion/AbstractIngestionEngine.js

```diff
@@ -1,66 +1,120 @@
 import { spawn } from 'child_process'
+import {
+	pipeline,
+	Writable,
+	Transform,
+} from 'stream'
```

Comment:
We should think about using minipass in the future. Here's [why](https://www.npmjs.com/package/minipass#differences-from-nodejs-streams).

Reply:
I'll spin up an issue for this as part of merging.